Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -303,9 +304,12 @@ void SetLastTransform(const CGAffineTransform* transform) {
         return &lastTransform;
     }
 
-    void AllowAllExceptFinalEndFigure(bool allow) {
-        SetAllowsFigureCalls(allow);
-        geometrySink->AllowAllExceptFinalEndFigure(allow);
+    HRESULT AllowAllExceptFinalEndFigure(bool allow) {
+        HRESULT hr = SetAllowsFigureCalls(allow);
```

Comment:
nit: might just want to `RETURN_IF_FAILED` the first one and return the second to keep w/ the pattern we used in CG